res/values/strings.xml

```diff
@@ -1,31 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
-	<string translatable="false" name="help_url">https://github.com/sspanak/tt9/blob/master/docs/user-manual.md</string>
```

Comment:
What happened here? I don't see any real changes or are there any? If not, let's revert the file and keep the diff minimal.

Reply:
You know, I'll revert it in my branch, while cleaning up the rest of the code.